lib/generators/arclight/templates/catalog_controller.rb

```diff
@@ -236,7 +236,7 @@ class CatalogController < ApplicationController
     end
 
     # These are the parameters passed through in search_state.params_for_search
-    config.search_state_fields += %i[id group hierarchy_context original_document]
+    config.search_state_fields += %i[group hierarchy hierarchy_context id nest_path original_document]
```

Comment:
Do we know why nest_path has to be in the search state?

Reply:
Looks like it's used to see if the document is in the current hierarchy: https://github.com/projectblacklight/arclight/blob/main/app/components/arclight/document_collection_context_component.rb#L42-L48

Reply:
I don't think they need to be in search state if we're just plucking them out of params directly, but I guess there's not a way to let Rails know we don't care 🤷‍♂️

Reply:
Right, so these have nothing to do with the "search state", they are only used for display options.

Reply:
@cbeer are you thinking something like https://github.com/projectblacklight/arclight/pull/1265 instead?